src/pages/Donate/DonateSupportUs.vue

```diff
@@ -1,251 +0,0 @@
-<template>
-	<www-page id="support-us-landing">
-		<div class="row page-content">
-			<div class="small-12 large-7 columns donation-form-holder">
-				<h1 class="donate-headline" v-html="headlineCopy"></h1>
-				<div class="donate-subhead" v-html="subHeadCopy"></div>
-				<donate-form
-					:key="1"
-					:button-text="buttonText"
-					:data="donationValues"
-					:selected-amount-setting="selectedAmountSetting"
-					:form-disclaimer="formDisclaimer"
-					:show-disclaimer="false"
-					:activate-monthly-option="true"
-				/>
-			</div>
-			<div class="small-12 large-5 columns donation-meter-holder">
-				<donate-support-us-right-rail :content="donationCalloutsCG" />
-			</div>
-			<div class="small-12 columns donation-faq-holder">
-				<kv-frequently-asked-questions
-					:headline="frequentlyAskedQuestionsHeadline"
-					:questions="frequentlyAskedQuestions"
-				/>
-			</div>
-		</div>
-	</www-page>
-</template>
-
-<script>
-import { gql } from '@apollo/client';
-
-import WwwPage from '@/components/WwwFrame/WwwPage';
-import KvFrequentlyAskedQuestions from '@/components/Kv/KvFrequentlyAskedQuestions';
-import { processPageContent } from '@/util/contentfulUtils';
-import DonateForm from '@/pages/Donate/DonateForm';
-import DonateSupportUsRightRail from '@/pages/Donate/DonateSupportUsRightRail';
```

Comment:
Looks like we can remove this component?

Reply:
+1